src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -94,6 +97,8 @@ public function load($resource, $type = null)
                     $controller = $this->parser->parse($controller);
                 } catch (\Exception $e) {
                     // unable to optimize unknown notation
+                } catch (\Throwable $e) {
+                    // unable to optimize unknown notation
```

Comment:
Makes sense, having it more specific and only throw away something, not everything seems more sane :+1: